lib/ui/widgets/gallery.dart

```diff
@@ -181,7 +186,7 @@ class _GalleryState<C extends GalleryCubit> extends State<Gallery>
 
         return PhotoViewGalleryPageOptions.customChild(
           child: child,
-          minScale: PhotoViewComputedScale.contained * 0.8,
+          minScale: PhotoViewComputedScale.contained * 1,
```

Comment:
Is there a reason this was changed? I cannot test this atm, I will test next week

Reply:
With 0.8, you can zoom out the image from its default position. Not just 'pinch' it, but after that it will suddenly not take the full width up anymore, which we thought was pretty cursed. Putting it at 1 instead solves that so fully zoomed out is still the full screen width.